.jenkins/pytorch/win-build.sh

```diff
@@ -78,24 +82,27 @@ set DISTUTILS_USE_SDK=1
 
 set CMAKE_GENERATOR=Ninja
 
-set NO_CUDA=1
-
-python setup.py install
+if "%REBUILD%"=="" (
+  set NO_CUDA=1
+  python setup.py install
+  if %errorlevel% neq 0 exit /b %errorlevel%
```

Comment:
This one is not acceptable here. You could use something like Delayed Expansion, some more details [here](https://ss64.com/nt/delayedexpansion.html).